Documentation/guides/BuildProcess.md

```diff
@@ -156,7 +156,7 @@ an [MSBuild PropertyGroup element](http://msdn.microsoft.com/en-us/library/t4w15
     debuggable.
 
     - **AndroidGenerateLayoutBindings** &ndash; Enables generation of [layout code-behind](LayoutCodeBehind.md)
-	  if set to `true` (the default) or disables it completely if set to `false`
+	  if set to `true` or disables it completely if set to `false` (the default)
```

Comment:
I wonder if the default value should be more explicitly called out, as is done for e.g. **AndroidFastDeploymentType**, a'la:

```markdown
if set to `true` or disables it completely if set to `false`.

The default value is `false`.
```

Reply:
Sounds good to me. I like switching out the parenthetical comment to have its own sentence. I have updated the PR accordingly.